src/lib/rank_utils.ts

```diff
@@ -22,9 +22,74 @@ interface IRankInfo {
     label: string;
 }
 
-export const MaxRank: number = 36;
+class Rating {
+    unset:boolean;
+    rating:number;
+    deviation:number;
+    volatility:number;
+    rank:number;
+    rank_label:string;
+    partial_rank:number;
+    partial_rank_label:string;
+    rank_deviation_labels:Array<string>;
+    rank_deviation:number;
+}
+
+export const MaxRank: number = 39;
+
+
+const MIN_RATING = 100;
+const MAX_RATING = 6000;
+
+export function rank_to_rating(rank:number) {
+    return 900 * Math.exp(0.032 * rank);
```

Comment:
I'm trying to see what the difference in ratings for players 1 rank apart is. I get rank_to_rating(30) = 2350 and rank_to_rating(31) = 2426. So the rating difference is 76. That seems low? I thought the glicko2 scale is the same as Elo? 

KGS ranges from 148 Elo per rank in the kyus to 226 Elo per rank in the dans. I think EGF is similar.